packages/db/drizzle/0032_hot_swordsman.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE `check` (
```

Comment:
Are we storing the checks in the db?

Reply:
Yes but not doing anything with them yet

Reply:
Yep we are storing the request on the db not doing anything with it though 
but storing some data,